src/v3/test/integration/authenticator-verification-email.test.tsx

```diff
@@ -167,7 +167,7 @@ describe('Email authenticator verification when email magic link = undefined', (
         },
         widgetOptions: { features: { autoFocus: true } },
       });
-      await findByText(/Verify with your email/);
+      await waitFor(() => findByText(/Verify with your email/));
```

Comment:
Does changing the global `asyncUtilTimeout` config not affect `findBy...` queries?

> findBy methods are a combination of getBy* queries and waitFor. They accept the waitFor options as the last argument (i.e. await screen.findByText('text', queryOptions, waitForOptions))
https://testing-library.com/docs/queries/about/#types-of-queries

Reply:
You are right. Reverted changes like this.